src/meta/protos/proto/udf.proto

```diff
@@ -49,8 +49,21 @@ message UDFScript {
   string runtime_version = 6;
 }
 
+message UDAFScript {
+  uint64 ver = 100;
+  uint64 min_reader_ver = 101;
+
+  string code = 1;
+  string language = 2;
+  string runtime_version = 3;
+  DataType return_type = 4;
+  repeated DataType arg_types = 5;
+  map<string, DataType> state_fields = 6;
+}
 
 message UserDefinedFunction {
+  reserved 7; // reserved for upcoming UDAFServer
```

Comment:
let's set `UDAFScript udaf_script = 7;` and update the tests.

We may not need `udaf server` in the future.